docs/ci.md

```diff
@@ -0,0 +1,213 @@
+# CI Tests
+
+OVN-Kubernetes runs the
+[Kubernetes E2E](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md)
+as CI. Github triggers are used to run a subset of the Kubernetes E2E tests on
+each pull request. Below describes how to update the set of tests that run and
+how to run these tests locally.
+
+## Updating CI test suite
+
+The tests that are run in CI are from the generated file:
```

Comment:
@Billy99 you can replace this with the following:

```markdown
The tests are run using [GitHub Actions](https://help.github.com/en/actions)
The workflow that describes the test run is located [here](ovn-kubernetes/.github/workflows/test.yml)

It performs the following tasks:
- Builds OVN-Kubernetes
- Checks out the Kubernetes source tree and compiles some dependencies
- Installs KIND
- Runs a matrix of End-To-End Tests using KIND
```

Reply:
Or rather add that to the previous section and for updating it's just `add a new matrix item to the test.yml file`

Reply:
Thanks. Done.